OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This would be adding a BOM. VS adds them in automatically any time you touch these files, so this makes in consistent with the rest at least.